### ISSUE TITLE
Cannot run tests with Gradle

### ISSUE THREAD

**Issue by daDangminh** [nus-cs2103-AY1819S2/forum#20]
I'm doing a test run on AB4 and while I could pass all tests using IntelliJ, I couldn't quite seem to run tests with Gradle. It looks like it has problem importing javafx. Can anyone help me?

![image](https://user-images.githubusercontent.com/25327972/52514498-4b5cfd00-2c4d-11e9-832c-ee54e55cd8a2.png)

**Comment by kylase-learning** [comment]
Are you running Java 9?

**Comment by daDangminh** [comment]
yes I am. In fact initially I was using JDK11, but afterwards I changed to JDK9 in project structures.

**Comment by kylase** [comment]
But it seems like you are running it off command prompt and not IntelliJ. Setting you Java version in IntelliJ does not mean you have set the system wide Java version which is determined by the `PATH` environment variable.

**Comment by kylase-learning** [comment]
But it seems like you are running it off command prompt and not IntelliJ. Setting the Java version in IntelliJ does not mean you have set the system wide Java version which is determined by the `PATH` environment variable.

You should check your Java version in command prompt first.

**Comment by daDangminh** [comment]
oh right that makes so much more sense. I'll try later!

**Comment by daDangminh** [comment]
So I am now able to run tests with Gradle, but it failed 4 tests out of 338 tests despite me not having made any changes to the code. What could this possibly mean?

![image](https://user-images.githubusercontent.com/25327972/52527034-3a6bc480-2cfd-11e9-89de-21eb59f1a35c.png)

**Comment by okkhoy** [comment]
@daDangminh take a look at this: https://github.com/nus-cs2103-AY1819S2/addressbook-level4/commit/231b72bc243f6cf6d9936c11c7ae798ea7b2f4c9
